cmake/compiler/gcc/compiler_flags.cmake

```diff
@@ -177,6 +177,9 @@ set_compiler_property(PROPERTY sanitize_address -fsanitize=address)
 
 set_compiler_property(PROPERTY sanitize_undefined -fsanitize=undefined)
 
+# GCC compiler flag for turning off thread-safe initialization of local statics
+set_property(TARGET compiler-cpp PROPERTY no_threadsafe_statics "-fno-threadsafe-statics")
```

Comment:
May I ask you to add the same property configuration for MWDT toolchain?
MWDT supports `-fno-threadsafe-statics` flag, so it would be just a copy of this line
```
set_property(TARGET compiler-cpp PROPERTY no_threadsafe_statics "-fno-threadsafe-statics")
```
 to `cmake/compiler/arcmwdt/compiler_flags.cmake` file.

Thanks!

Reply:
Added it to the file on the first commit, thank you for the feedback!

Reply:
Thanks!